### PR TITLE
get rid of  library json-simple (#732)

### PR DESCRIPTION
**Pull request by rudosch**
issue #732

### CODE REVIEW

**Comment by rudosch** [comment]
I'll have a look on the failed linter check asap

File: pom.xml [hunk 1]
```diff
@@ -247,12 +250,6 @@
 									</resources>
 								</configuration>
 							</execution>
-						</executions>
-					</plugin>
-					<plugin>
-						<artifactId>maven-resources-plugin</artifactId>
```
Comment (ismisepaul) [inline pom.xml:253]: Why is the resources plugin being removed?

File: src/main/java/servlets/module/challenge/CsrfChallengeTargetJSON.java [hunk 2]
```diff
@@ -61,7 +60,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     ResourceBundle csrfGenerics =
         ResourceBundle.getBundle("i18n.servlets.challenges.csrf.csrfGenerics", locale);
 
-    response.setContentType("text/html");
```
Comment (ismisepaul) [inline src/main/java/servlets/module/challenge/CsrfChallengeTargetJSON.java]: Should we not be setting the content type?

File: src/main/java/servlets/module/challenge/XxeChallenge1.java [hunk 3]
```diff
@@ -62,7 +62,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     // Setting IpAddress To Log and taking header for original IP if forwarded from proxy
     ShepherdLogManager.setRequestIp(request.getRemoteAddr(), request.getHeader("X-Forwarded-For"));
     log.debug(LEVEL_NAME + " Servlet Accessed");
-    response.setContentType("text/html");
```
Comment (ismisepaul) [inline src/main/java/servlets/module/challenge/XxeChallenge1.java]: Should we not set the content type?

File: pom.xml (hunk 1 above)
Comment (rudosch) [reply pom.xml:253]: Two executions that use same plugin are now combined. Avoids maven warnings.

File: src/main/java/servlets/module/challenge/CsrfChallengeTargetJSON.java (hunk 2 above)
Comment (rudosch) [reply src/main/java/servlets/module/challenge/CsrfChallengeTargetJSON.java]: You are right, actually there is no reason to remove this.

File: src/main/java/servlets/module/challenge/XxeChallenge1.java (hunk 3 above)
Comment (rudosch) [reply src/main/java/servlets/module/challenge/XxeChallenge1.java]: You are right, actually there is no reason to remove this.

**Comment by rudosch** [comment]
I will add another commit to restore setting of content type.

**Comment by rudosch** [comment]
I'm sorry, I can't find the linting error. Do you?
Shall I close this PR and start over with a new one, this time based on your recent commits?

**Comment by ismisepaul** [comment]
This is annoying alright, what I do is run the google-java-formatter tool to automatically do it for me and then push e.g. 

1) Download https://github.com/google/google-java-format/releases/tag/v1.15.0
2) Run `java -jar google-java-format-1.15.0-all-deps.jar  --replace <PATH-TO-PROJECT>/SecurityShepherd//**/*.java`

**Comment by rudosch** [comment]
Thanks for your hint, but I should have been more clear in my previous comment:
I already checked my PR commits (i.e. the files changed by those commits) exactly like you said and found no issues!
I used parameter -n to let the linter tell me which files will be affected by changes, but there were none.
(that's how I was able to fix the linter issues of my very first commit of this PR)
So I'll have another look at it tonight.

**Comment by rudosch** [comment]
I finally found the extra blank line that the linter didn't like ;-)

**Comment by ismisepaul** [comment]
Thanks @rudosch maybe the linting happens in the pipeline so people don't have to go through this headache...

I'm surprised the `--replace` flag when running google-java-format didn't work though (it's saved me a few headaches in the past)

That's 4 CVEs taken care of this week 🎉

**Pull request merged**